__init__.py

```diff
@@ -54,6 +54,7 @@ def _get_default_cycles():
 app.jinja_env.globals['api_location'] = config.api_location
 app.jinja_env.globals['api_version'] = config.api_version
 app.jinja_env.globals['api_key'] = config.api_key_public
+app.jinja_env.globals['debug'] = config.debug
```

Comment:
Doesn't seem like we have a use for it right now. Might as well take it out of the jinja globals until we need it.